ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -85,12 +85,14 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "http_2xx_failed_requests
   query = <<-QUERY
 requests
 ${local.skip_on_weekends}
-| where toint(resultCode) between (200 .. 299) and success == false and timestamp >= ago(5m)
+| where toint(resultCode) between (200 .. 299) and timestamp >= ago(5m)
+| sort by timestamp asc
+| summarize failureRate = iff((todouble(sumif(1, success == false)) * 100 / todouble(sumif(1, success == true)) >= ${var.http_2xx_failure_rate_threshold}), 1, 0)
```

Comment:
I'm pretty excited about how much I was able to cut down on this query, and it feels much easier to understand than the previous version.

If you're interested in viewing a chart of this data as I included in the description, you can do so with a very similar query.
```
requests
| where toint(resultCode) between (200 .. 299)
| sort by timestamp asc
| summarize failureRate = (todouble(sumif(1, success == false)) * 100 / todouble(sumif(1, success == true))) by bin(timestamp, 5m)
| render timechart 
```
I recommend selecting a custom timeframe from the dropdown in the Azure monitor but you can include you timeframe in the where clause.